.travis.yml

```diff
@@ -9,6 +9,7 @@ env:
   - TOXENV=py27,codecov
   - TOXENV=py27-functional,codecov
   - TOXENV=py35-functional,codecov
+  - TOXENV=docs,codecov
```

Comment:
do we need codecov for all of these? I think it will fail for doc or functional tests. Also we have a coverage env, maybe that is enough?